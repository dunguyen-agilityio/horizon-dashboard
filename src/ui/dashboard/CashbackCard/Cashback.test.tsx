import { render } from '@testing-library/react';

import Cashback from '.';

describe('TableCheck tests', () => {
  it('Should match snapshot', () => {
    const { container } = render(<Cashback />);
    expect(container).toMatchSnapshot();
  });
});
