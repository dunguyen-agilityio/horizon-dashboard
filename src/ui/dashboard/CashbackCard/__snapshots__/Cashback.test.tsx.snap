// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableCheck tests Should match snapshot 1`] = `
<div>
  <div
    class="w-full h-full sm:w-card md:w-auto bg-white dark:bg-indigo p-5 pb-[33px] rounded-md"
  >
    <div
      class="relative w-full h-[185px]"
    >
      <img
        alt="Starbucks Cover"
        data-nimg="fill"
        decoding="async"
        loading="lazy"
        sizes="100vw"
        src="/_next/image?url=%2Fstarbucks-cover.webp&w=3840&q=75"
        srcset="/_next/image?url=%2Fstarbucks-cover.webp&w=640&q=75 640w, /_next/image?url=%2Fstarbucks-cover.webp&w=750&q=75 750w, /_next/image?url=%2Fstarbucks-cover.webp&w=828&q=75 828w, /_next/image?url=%2Fstarbucks-cover.webp&w=1080&q=75 1080w, /_next/image?url=%2Fstarbucks-cover.webp&w=1200&q=75 1200w, /_next/image?url=%2Fstarbucks-cover.webp&w=1920&q=75 1920w, /_next/image?url=%2Fstarbucks-cover.webp&w=2048&q=75 2048w, /_next/image?url=%2Fstarbucks-cover.webp&w=3840&q=75 3840w"
        style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
      />
      <img
        alt="Starbucks Logo"
        class="absolute -bottom-[30px] left-[13px]"
        data-nimg="1"
        decoding="async"
        height="60"
        loading="lazy"
        src="/_next/image?url=%2Fstarbucks-logo.webp&w=128&q=75"
        srcset="/_next/image?url=%2Fstarbucks-logo.webp&w=64&q=75 1x, /_next/image?url=%2Fstarbucks-logo.webp&w=128&q=75 2x"
        style="color: transparent;"
        width="60"
      />
      <button
        class="p-[10px] rounded-xl absolute top-[14px] right-[14px] bg-gradient-gray backdrop:blur-[25px] disabled:cursor-default"
        disabled=""
      >
        <svg
          fill="none"
          height="24"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            clip-path="url(#clip0_201_2868)"
          >
            <path
              d="M12 2C6.5 2 2 6.5 2 12C2 17.5 6.5 22 12 22C17.5 22 22 17.5 22 12C22 6.5 17.5 2 12 2ZM12 20C7.59 20 4 16.41 4 12C4 7.59 7.59 4 12 4C16.41 4 20 7.59 20 12C20 16.41 16.41 20 12 20ZM12.5 7H11V13L16.2 16.2L17 14.9L12.5 12.2V7Z"
              fill="white"
            />
          </g>
          <defs>
            <clippath
              id="clip0_201_2868"
            >
              <rect
                fill="white"
                height="24"
                width="24"
              />
            </clippath>
          </defs>
        </svg>
      </button>
    </div>
    <h3
      class="text-primary dark:text-white text-1.5xl font-bold leading-xl pt-10"
    >
      Starbucks
    </h3>
    <div
      class="pt-[6px] flex items-center gap-[10px]"
    >
      <svg
        fill="none"
        height="24"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          clip-path="url(#clip0_201_2865)"
        >
          <path
            d="M21.9 5H18V2C18 1.45 17.55 1 17 1C16.45 1 16 1.45 16 2V5H12.1C11.51 5 11.05 5.51 11.1 6.1L11.22 7.31C14.9 8.16 18 10.77 18 15L18.02 23H19.72C20.56 23 21.25 22.35 21.35 21.53L22.89 6.1C22.95 5.51 22.49 5 21.9 5ZM15 21H2C1.45 21 1 21.45 1 22C1 22.55 1.45 23 2 23H15C15.55 23 16 22.55 16 22C16 21.45 15.55 21 15 21ZM2.1 15H14.9C15.52 15 16.01 14.44 15.89 13.84C15.24 10.61 11.87 8.99 8.5 8.99C5.13 8.99 1.77 10.61 1.11 13.84C0.99 14.44 1.49 15 2.1 15ZM15 17H2C1.45 17 1 17.45 1 18C1 18.55 1.45 19 2 19H15C15.55 19 16 18.55 16 18C16 17.45 15.55 17 15 17Z"
            fill="#A3AED0"
          />
        </g>
        <defs>
          <clippath
            id="clip0_201_2865"
          >
            <rect
              fill="white"
              height="24"
              width="24"
            />
          </clippath>
        </defs>
      </svg>
      <span
        class="text-secondary dark:text-secondary leading-xl text-lg font-bold"
      >
        10% cashback & off
      </span>
    </div>
  </div>
</div>
`;
